internal/form/names.go

```diff
@@ -1,9 +1,31 @@
 package form
 
-var FieldNames = Names{
-	LanguagePreference: "language-preference",
+var FieldNames = SharedFieldNames{
+	LanguagePreference: LanguagePreferenceFieldNames{
```

Comment:
I think this would be ok as it was, since it doesn't have multiple fields (and I don't like the repetition it has introduced)